src/components/Trade/Components/TradeQuotes/TradeQuote.tsx

```diff
@@ -98,20 +98,25 @@ export const TradeQuoteLoaded: React.FC<TradeQuoteLoadedProps> = ({
   )
   const updateFees = useSwapperStore(state => state.updateFees)
   const updateTradeAmountsFromQuote = useSwapperStore(state => state.updateTradeAmountsFromQuote)
+  const setSwapperDefaultAffiliateBps = useSwapperStore(
+    state => state.setSwapperDefaultAffiliateBps,
+  )
 
   const handleSwapperSelection = useCallback(
     (activeSwapperWithMetadata: SwapperWithQuoteMetadata) => {
       updateActiveSwapperWithMetadata(activeSwapperWithMetadata)
+      setSwapperDefaultAffiliateBps()
```

Comment:
See question in the action definition, I believe view-layer shouldn't need to handle this, this simply calls a selector with the root state to set `affiliateBps` so I believe we don't set anything would work the same

Reply:
See https://github.com/shapeshift/web/pull/4353#discussion_r1181363035, removed in https://github.com/shapeshift/web/pull/4353/commits/b90024ea6dd4e11c94af84f00347767d56639bfd 👌